test/Sema/accessibility_shared_private.swift

```diff
@@ -0,0 +1,76 @@
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: %utils/split_file.py -o %t %s
+// RUN: %target-swift-frontend -swift-version 4 -typecheck %t/declarations.swift %t/other_file_extensions.swift -verify
```

Comment:
Can you also run this test in multi-file mode, using `-primary-file` before each file in turn? Sometimes that behaves differently from whole-module mode (which is the default when invoking `%target-swift-frontend` directly).